core/tauri/src/image/mod.rs

```diff
@@ -186,20 +188,39 @@ impl TryFrom<Image<'_>> for tray_icon::Icon {
   }
 }
 
+/// The counterpart of the `@tauri-apps/api/image` Image class.
```

Comment:
This is not exactly true though, I believe this is equivalent to `string | number[] | Unit8Array | ArrayBuffer | Image` so maybe needs to change the wording and mention `transformImage`?